tests/test_config_gsobject.py

```diff
@@ -595,6 +595,7 @@ def test_realgalaxy():
                  },
         'gal5' : { 'type' : 'RealGalaxy' , 'index' : 23, 'noise_pad_size' : 10 },
         'gal6' : { 'type' : 'RealGalaxyOriginal' },
+        'gal7' : { 'type' : 'RealGalaxy' , 'random' : True}
```

Comment:
Yes, that's exactly what I was thinking of doing - a set of new tests for COSMOSGalaxy.  Will do shortly and post when done adding unit tests.